api/WebGL2RenderingContext.json

```diff
@@ -715,7 +854,7 @@
       },
       "compressedTexImage3D": {
         "__compat": {
-          "mdn_url": "https://developer.mozilla.org/docs/Web/API/WebGL2RenderingContext/compressedTexImage3D",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/WebGL2RenderingContext/compressedTexImage2D",
```

Comment:
This is an incorrect change. It should be https://developer.mozilla.org/docs/Web/API/WebGL2RenderingContext/compressedTexImage3D (3D instead of 2D)